### PR TITLE
Makefile and test data generator

### PR DESCRIPTION
**Pull request by rajivsam**
This branch has the following changes:

1. Makefile to build python package
2. Python script to simulate datasets arriving every month. Models are built every month and metadata generated from model building is updated in the Arangopipe database.

### CODE REVIEW

File: arangopipe/tests/test_data_generator/generate_model_data.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    if clean:
+        admin = ArangoPipeAdmin()
+        admin.delete_arangomldb()
+        admin.create_db()
+        admin.create_enterprise_ml_graph()
+        proj_info = {"name": "Home_Value_Assessor"}
+        proj_reg = admin.register_project(proj_info)
+    
+    the_period = period_string_generator()
```
Comment (joerg84) [inline arangopipe/tests/test_data_generator/generate_model_data.py]: s/ the_period/period?

**Comment by joerg84** [comment]
@rajivsam Could we (next time) split such changes in multiple PRs? One for makefile, one for testgen, ...

File: arangopipe/tests/test_data_generator/generate_model_data.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Jul 12 07:25:18 2019
+
```
Comment (joerg84) [inline arangopipe/tests/test_data_generator/generate_model_data.py]: Next time we should add the Apache 2 license, I will create a PR for the existing files

File: arangopipe/makefile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+SRC = arangopipe/*.py
+python_arangopipe:$(SRC)
+	python3 setup.py sdist bdist_wheel
+clean:
+	rm -rf arangopipe.egg-info build dist
+	
+
```
Comment (joerg84) [inline arangopipe/makefile]: Unnecessary blank file.

File: arangopipe/tests/test_data_generator/generate_model_data.py (hunk 1 above)
Comment (rajivsam) [reply arangopipe/tests/test_data_generator/generate_model_data.py]: sure.

File: arangopipe/tests/test_data_generator/generate_model_data.py (hunk 2 above)
Comment (rajivsam) [reply arangopipe/tests/test_data_generator/generate_model_data.py]: sure. I will make sure I include the Apache 2 license in the data generators for hyper-parameters and exploratory data anlaysis.

File: arangopipe/makefile (hunk 3 above)
Comment (rajivsam) [reply arangopipe/makefile]: Not sure I follow. Which file is an uncessary blank file - the makefile? This builds the arangopipe python package.

**Pull request merged**